tutorial_src/gue/prog/guechk.cpp

```diff
@@ -0,0 +1,58 @@
+#include <bits/stdc++.h>
+#include "oi.h"
+using namespace oi;
```

Comment:
Nie wiem czy pochwalam używanie usingów, po to to jest w osobnych namespacach żeby widać było skąd wzięło się jaką funkcję. Biblioteka bits też tak średnio mi się podoba, ale to już mniejszy problem (imo dopuszczalna w solvach, ale nie w programach).

Reply:
Ups, bitsy to już odruch bezwarunkowy, poprawię

Namespace'a oi mogę jeszcze wyrzucić, bo faktycznie uwaga tu jest słuszna, ale std bym zostawił, bo to jest standardowe